crates/goose/src/agents/snapshots/goose__agents__prompt_manager__tests__one_extension.snap

```diff
@@ -39,20 +39,12 @@ and platform__list_resources on this extension.
 how to use this extension
 
 
-# LLM Tool Selection Instructions
-    Important: the user has opted to dynamically enable tools, so although an extension could be enabled, \
-    please invoke the llm search tool to actually retrieve the most relevant tools to use according to the user's messages.
-    For example, if the user has 3 extensions enabled, but they are asking for a tool to read a pdf file, \
-    you would invoke the llm_search tool to find the most relevant read pdf tool.
-    By dynamically enabling tools, you (goose) as the agent save context window space and allow the user to dynamically retrieve the most relevant tools.
-    Be sure to format a query packed with relevant keywords to search for the most relevant tools.
-    In addition to the extension names available to you, you also have platform extension tools available to you.
-    The platform extension contains the following tools:
-    - search_available_extensions
-    - manage_extensions
-    - read_resource
-    - list_resources
-    
+Important: the user has opted to dynamically enable tools, so although an extension could be enabled,
```

Comment:
I'm not sure about these instructions. Shouldn't we just advertise all the tools and then include a more minimal message something like: 

"if you don't find anything relevant in the standard enabled tools you can search for others that may be relevant via the `llm_search` tool and then dynamically enable an extension that has that tool"